packages/v4/src/content/design-guidelines/usage-and-behavior/toolbar/toolbar.md

```diff
@@ -9,147 +9,126 @@ Toolbars can be included to allow a user to manage and manipulate a data set. Da
 
 ## Elements
 
-The toolbar is a container for holding a variety of elements (toolbar items and groups) that are used to manipulate a set of data.
+The toolbar is a container for holding a variety of elements (toolbar items and groups) that are used to manipulate a set of data. All items should be centered vertically within the toolbar. There is a 24px of padding before and after the toolbar content.
```

Comment:
```suggestion
The toolbar is a container for holding a variety of elements (toolbar items and groups) that are used to manipulate a set of data. All items should be centered vertically within the toolbar. There is 16px of horizontal padding before and after the toolbar content.
```

Reply:
I've identified the padding values as a potential enhancement. Currently, the padding left/right is `16px`, but I believe we should update this padding to be consistent with `pf-c-page__main-section`, `__card`, etc
https://github.com/patternfly/patternfly/issues/3240

Reply:
Yes, I do remember a discussion about adjusting this. So I guess that means the toolbar examples will also need adjusting to reduce the spacing on either end. @gdoyle1 that will then also apply to the Sketch Library which has 24px spacers.

@mattnolting can you confirm this before we make those updates?

Reply:
Based on our conversation yesterday, I just deleted this statement.